### ISSUE TITLE
[Bug]: Unstable testcase

### ISSUE THREAD

**Issue by jiaoew1991** [milvus-io/milvus#20640]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Environment

```markdown
- Milvus version: master
- Deployment mode(standalone or cluster): standalone
- SDK version(e.g. pymilvus v2.0.0rc2): ci 
- OS(Ubuntu or CentOS): Ubuntu
- CPU/Memory: 
- GPU: 
- Others:
```

### Current Behavior

unstable case https://jenkins.milvus.io:18080/blue/organizations/jenkins/Milvus%20HA%20CI/detail/PR-20621/9/pipeline/

### Expected Behavior

make it stable

**Comment by yah01** [comment]
https://jenkins.milvus.io:18080/blue/organizations/jenkins/Milvus%20HA%20CI/detail/PR-20806/1/pipeline/142

**Comment by yah01** [comment]
fixed #20866
